cfme/fixtures/embedded_ansible.py

```diff
@@ -0,0 +1,99 @@
+import pytest
+from wrapanapi.utils.random import random_name
+
+
+def instantiate_namespace(domain_name, namespace_levels):
+    namespace = domain_name
+    for namespace_name in namespace_levels:
+        namespace = namespace.namespaces.instantiate(namespace_name)
+    return namespace
+
+
+def copy_ae_instance(appliance, instance_fqdn, domain_name):
+    """
+    Copy AE Instance into a domain. We split 'instance_fqdn' which is a full string path to existing
+    AE instance. Then based on the values we get from split, proper classes are instantiated.
+    AE Instance is copied to provided domain name, then instantiated and returned under new domain.
+    Example:
+    if 'instance_fqdn' is 'ManageIQ/System/Event/EmsEvent/Nuage/nuage_enterprise_create' we get the
+    following values:
+    source = ['ManageIQ', 'System', 'Event', 'EmsEvent', 'Nuage', 'nuage_enterprise_create']
+    'ManageIQ' as source_domain
+    'nuage_enterprise_create' as source AE instance
+    'Nuage' as source AE class
+    the remaining strings left after 'pop' calls are the AE namespaces which are instantiated
+    through 'instantiate_namespace' function
+    AE instance is then copied to a new domain. We also instantiate all the destination classes so
+    we get proper parent-child structure.
+
+
+    :param appliance: appliance
```

Comment:
Thanks so much for adding this docblock content, this is great and lays out your intent very well.

Please use sphynx-napoleon style for these docblocks.
https://cfme-tests.readthedocs.io/en/master/guides/documenting.html?highlight=napoleon

This section should look more like:
```
Args:
    appliance: Appliance object
    instance_fqdn (str): full path <yada>
```

Reply:
Updated :)